src/PointwiseFunctions/AnalyticSolutions/Elasticity/HalfSpaceMirror.cpp

```diff
@@ -183,36 +181,26 @@ tuples::TaggedTuple<Tags::Strain<3>> HalfSpaceMirror::variables(
                "smaller.\n");
     }
   }
-  return {std::move(strain)};
 }
 
-tuples::TaggedTuple<::Tags::FixedSource<Tags::Displacement<3>>>
-HalfSpaceMirror::variables(
-    const tnsr::I<DataVector, 3>& x,
-    tmpl::list<::Tags::FixedSource<Tags::Displacement<3>>> /*meta*/) noexcept {
-  return {make_with_value<tnsr::I<DataVector, 3>>(x, 0.)};
+template <typename DataType>
+void HalfSpaceMirrorVariables<DataType>::operator()(
```

Comment:
Same question about the replacement of variables specialization with operator() in a detail class design you've chosen